kivy/input/providers/mouse.py

```diff
@@ -159,11 +174,34 @@ def __init__(self, device, args):
                 self.disable_on_activity = True
             elif arg == 'disable_multitouch':
                 self.disable_multitouch = True
+            elif arg == 'disable_hover':
+                self.disable_hover = True
             elif arg == 'multitouch_on_demand':
                 self.multitouch_on_demand = True
             else:
                 Logger.error('Mouse: unknown parameter <%s>' % arg)
 
+    def _get_disable_hover(self):
+        return self._disable_hover
+
+    def _set_disable_hover(self, value):
+        if self._disable_hover != value:
+            if self._running:
+                if value:
+                    self._stop_hover_events()
+                else:
+                    self._start_hover_events()
+            self._disable_hover = value
+
+    disable_hover = property(_get_disable_hover, _set_disable_hover)
+    '''Disables dispatching of hover events if set to `True`.
+
+    .. versionadded:: 2.1.0
+
+    :attr:`disable_hover` is a :class:`property` instance and defaults to
+    `False`.
```

Comment:
```suggestion
    '''Disables dispatching of hover events if set to ``True`` (default).

    .. versionadded:: 2.1.0
```

There's no need to say it's a `property`, we only do that for kivy style properties because otherwise it's not visible in the docs. But for normal properties, it's not very important.

Reply:
Ok 👍